app/dashboard/models.py

```diff
@@ -176,6 +176,7 @@ class Bounty(SuperModel):
     PERMISSION_TYPES = [
         ('permissionless', 'permissionless'),
         ('approval', 'approval'),
+        ('reserved', 'reserved'),
```

Comment:
should we migrate existing bounties who are reserved to the new permission type?

Reply:
@owocki what would you say is the best effective way to do this? I would like to just put relevant code in migration file, but we can also write a management command to do the same. @francopetra @thelostone-mc

Reply:
write a management command and/or a migration to do it.  migration will be run automatically, so id prefer that.  if mgmt command, we just need to remind @danlipert to run it in prod after he deploys

Reply:
@francopetra @owocki @SaptakS A new migration would be ideal :+1:

Reply:
@danlipert @owocki Hey, as I understand I should delete this, and already created a migration in app/dashboard/migrations/0001_initial.py (Line 94). 
Not sure how to migrate existing ones though.

Reply:
@francopetra we're talking a bout a new migration as described here: https://docs.djangoproject.com/en/2.2/topics/migrations/ with a custom forwards method: https://docs.djangoproject.com/en/2.2/howto/writing-migrations/